Lib/test/test_tempfile.py

```diff
@@ -848,6 +848,12 @@ def test_for_tempdir_is_bytes_issue40701_api_warts(self):
         finally:
             tempfile.tempdir = orig_tempdir
 
+    def test_absolute_path(self):
+        """Test that the path returned is absolute"""
+        path = tempfile.mkdtemp(dir=".")
+        self.assertEqual(os.path.isabs(path), True)
+        os.rmdir(path)
```

Comment:
If the line before this one fails for any reason, then the file will not be cleaned up.  Look at how other methods handle this cleanup!

Reply:
Done in ff507d9002